app/javascript/javascripts/modules/datepicker.js.coffee

```diff
@@ -30,12 +30,14 @@ class app.Datepicker
     maxDate = if input.attributes.maxdate? then new Date(input.attributes.maxdate.value) else null
 
     options = $.extend({ onSelect: (d, i) -> self.track(this, d, i) },
+      $.datepicker.regional[$('html').attr('lang')],
```

Comment:
The language in `[$('html').attr('lang')]` is set to "fr", so I tried to hard-code it as `fr_CH` but it didn't work. So apparently `datepicker` doesn't support it.

Reply:
Ah after importing `datepicker-fr-CH` it works with `fr-CH`:
```diff
diff --git a/app/javascript/packs/application.js b/app/javascript/packs/application.js
index 3bbcce375..e09d1031e 100644
--- a/app/javascript/packs/application.js
+++ b/app/javascript/packs/application.js
@@ -29,6 +29,7 @@ import 'moment'; // used by events/date_period_validator.js.coffee
 import 'jquery-ui/ui/widgets/datepicker';
 import 'jquery-ui/ui/i18n/datepicker-de';
 import 'jquery-ui/ui/i18n/datepicker-fr';
+import 'jquery-ui/ui/i18n/datepicker-fr-CH';
 import 'jquery-ui/ui/i18n/datepicker-it';
 import 'jquery-ui/ui/effects/effect-highlight';
 ```